packages/mermaid/src/logger.ts

```diff
@@ -2,7 +2,7 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
 /* eslint-disable @typescript-eslint/no-empty-function */
 /* eslint-disable no-console */
-import dayjs from 'dayjs';
+import dayjs from 'dayjs/esm/index.js';
```

Comment:
Unfortunately, I think `dayjs`'s ESM support is still a bit lackluster: https://github.com/iamkun/dayjs/issues/1765

We've previously tried to use `dayjs/esm`, but had to revert it due to dayjs's ESM support being non-standard, and so many users faced issues with it: https://github.com/mermaid-js/mermaid/issues/4438